ponyup-init.sh

```diff
@@ -106,6 +107,17 @@ if [ "${response}" = "[]" ]; then
   exit 1
 fi
 
+ponyup_pkg="$(json_field "${response}" version)"
+ponyup_pkg="${ponyup_pkg}-${platform_triple_cpu}-${download_os##*-}"
+
+if [ "${repository}" = releases ]; then
+  ponyup_pkg="ponyup-release-${ponyup_pkg}"
+else
+  ponyup_pkg="ponyup-nightly-${ponyup_pkg}"
+fi
+echo "pkg: ${ponyup_pkg}"
+echo "${ponyup_pkg}" >"${ponyup_root}/.lock"
```

Comment:
i personally always do a space before and after ">"

Reply:
This was done automatically by my editor. will fix